browser/ai_chat/upload_file_helper_unittest.cc

```diff
@@ -21,13 +22,30 @@
 #include "content/public/test/file_system_chooser_test_helpers.h"
 #include "content/public/test/test_renderer_host.h"
 #include "content/public/test/web_contents_tester.h"
+#include "gmock/gmock.h"
```

Comment:
nit: remove

Reply:
thanks I will remove it, it was automatically added by my clangd setup in vim

Reply:
4ca593b6983ed832e4fa756029a974bfd2c89f81